src/delay.rs

```diff
@@ -1,6 +1,8 @@
 //! Delays
 
 use cast::u32;
+use core::convert::Infallible;
```

Comment:
I'm pretty sure this is not the correct error to use for functions that can't fail (or which proper error detection has yet to be implemented). Any suggestion?

Reply:
this is the correct marker to use when defining infallible implementations (as in, `type Error = Infallible;` in a trait implementation), where consuming functions they need to be generic over an error type.

Reply:
Thanks for the input!